src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Color randomColor = colorSupplier.getRandomColor();
```

Comment:
Introduce `Random` and `ColorSupplier` as field variables. You don't need to create new objects every time you call `getRandomBall`